apps/web/src/app/api/projects/[projectId]/commits/[commitUuid]/documents/[documentUuid]/evaluations/[evaluationId]/evaluation-results/average/route.ts

```diff
@@ -5,6 +5,7 @@ import { authHandler } from '$/middlewares/authHandler'
 import { errorHandler } from '$/middlewares/errorHandler'
 import { NextRequest, NextResponse } from 'next/server'
 
+// FIXME: Use generic types. Check other routes for examples.
```

Comment:
Another great thing is that fully typing all the routes in the web API doesn't need to be done now. We can progressively do it. Good boy scout mentality.